### PR TITLE
Master

### PR DESCRIPTION
**Pull request by karumanchi1116**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -24,4 +24,4 @@ ESHOP_PROD_EXTERNAL_DNS_NAME_OR_IP=10.121.122.92
 #ESHOP_SERVICE_BUS_USERNAME=<ServiceBusUserName-OnlyUsedIfUsingRabbitMQUnderwindows>
 #ESHOP_SERVICE_BUS_PASSWORD=<ServiceBusUserPassword-OnlyUsedIfUsingRabbitMQUnderwindows>
 #INSTRUMENTATION_KEY=
-#USE_LOADTEST=<True/False>
\ No newline at end of file
+#USE_LOADTEST=<True/False>
```
Comment (jzeferino) [inline .env]: This was an unnecessary change to commit. nothing was changed.

File: src/Mobile/README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 # eShopOnContainers
 
-eShopOnContainers is a reference app whose imagined purpose is to serve the mobile workforce of a fictitious company that sells products. The app allow to manage the catalog, view products, manage the basket and the orders.
+eShopOnContainers is a reference app whose imagined purpose is to serve the mobile workforce of a fictitious company that sells products. The app would allow a user to manage their catalog, view products, manage the basket/cart and the orders.
```
Comment (jzeferino) [inline src/Mobile/README.md]: Suggestion:

> The app would **allows the user** to manage their catalog, view products, manage the basket/cart and the orders.

File: src/Mobile/README.md [hunk 3]
```diff
@@ -97,7 +97,7 @@ The app has the following screens:
 
 * a auth screen
 * a catalog list
-* a profile section with a order list
```
Comment (jzeferino) [inline src/Mobile/README.md]: Suggestion:

> a profile section with a list of orders

File: src/Mobile/README.md [hunk 4]
```diff
@@ -142,7 +142,7 @@ The application performs the deployment and even boots into the emulator, but st
 
 The error is related with incompatibilities between the host processor and the Hyper-V virtual machine.
 
-In Windows 10, we press the start button and write MMC. Next, click the Hyper-V Manager option:
+In Windows 10, we press the start button and write MMC, short for Microsoft Management Console. Next, click the Hyper-V Manager option:
```
Comment (jzeferino) [inline src/Mobile/README.md]: Suggestion:

> In Windows 10, ~~we~~ press the start button and write MMC, short for Microsoft Management Console. Next, click the Hyper-V Manager option:

**Review by jzeferino — CHANGES_REQUESTED** [review]

**Comment by mvelosop** [comment]
Hi @karumanchi1116, thanks for your contributions, but we don't accept PRs to **master**, as per https://github.com/dotnet-architecture/eShopOnContainers/blob/master/branch-guide.md.

If you want to submit the PR, please fork the repo in GitHub and create the new branch from the current **dev**.

Please keep in mind that you must also sign the Contributor License Agreement (CLA) to accept th PR.

Thanks.

**Pull request closed without merging**